azure-devops/azext_devops/dev/common/telemetry.py

```diff
@@ -67,7 +67,9 @@ def _is_telemetry_enabled():
     az_config = az_cli_ctx.config
     if az_config.has_option('core', 'collect_telemetry'):
         collect_telemetry = az_config.get('core', 'collect_telemetry')
-    return bool(collect_telemetry is None or collect_telemetry != 'no')
+    elif az_config.has_option('defaults', 'collect_telemetry'):
+        collect_telemetry = az_config.get('defaults', 'collect_telemetry')
+    return bool(collect_telemetry is None or collect_telemetry not in ('no', 'false', 'off', '0'))
```

Comment:
Would it not make sense to use the `getboolean` method here? [Link](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean).

I'm the guy who originally reported that the telemetry disabling wasn't working, and I've no insight into this config object, so if it turns out to not be a `ConfigParser` object, and I'm way off base I apologize. The reason I bring it up is that I saw in one of the other Azure CLI repos, that `getboolean` was being called.

Reply:
@aodj Your feedback is valid in terms of making the check consistent. 
We are still working on getting this fixed in the correct way, we will mostly work with Azure CLI or contribute there to have the right fix in place.